frontend/public/components/app-contents.tsx

```diff
@@ -704,7 +701,7 @@ const AppContents: React.FC<{}> = () => {
               path="/k8s/ns/:ns/pods/:podName/containers/:name"
               loader={() => import('./container').then((m) => m.ContainersDetailsPage)}
             />
-            <Route path="/k8s/ns/:ns/:plural/~new" exact component={CreateResource} />
```

Comment:
don't you have a different PR for that ?

Reply:
https://github.com/openshift/console/pull/10488 
Im hoping when ^^ will merged, this file will automatically be removed as it will be the same as master, if not once it merged ill remove it

Reply:
didnt waited, removed it